src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -460,6 +486,18 @@ public JdbcSinkConnectorConfig(Map<String, String> props) {
         this.databaseTimezone = config.getString(DATABASE_TIME_ZONE_FIELD);
         this.postgresPostgisSchema = config.getString(POSTGRES_POSTGIS_SCHEMA_FIELD);
         this.sqlServerIdentityInsert = config.getBoolean(SQLSERVER_IDENTITY_INSERT_FIELD);
+
+        String columnExcludeList = config.getString(COLUMN_EXCLUDE_LIST);
+        String columnIncludeList = config.getString(COLUMN_INCLUDE_LIST);
+
+        this.columnsFiltered = !(Strings.isNullOrEmpty(columnExcludeList) && Strings.isNullOrEmpty(columnIncludeList));
+
+        if (!Strings.isNullOrEmpty(columnExcludeList)) {
+            this.columnFilter = ColumnFilterFactory.createExcludeFilter(columnExcludeList);
+        }
+        else if (!Strings.isNullOrEmpty(columnIncludeList)) {
+            this.columnFilter = ColumnFilterFactory.createIncludeFilter(columnIncludeList);
+        }
```

Comment:
Hi @ani-sha, so I'd like to see this encapsulated just a bit deeper to avoid the `columnsFiltered` variable entirely.  

If you look at `Selectors#build()` in the main repository, you'll see that we provide the include/exclude filters, and from there, check if these have values or not.  If both are empty, it returns a `Predicate` that upon a call to match always returns true.  If include or exclude have a value, it prioritizes building the match `Predicate` against the include and then the exclude configuration.

This has the benefit that inside the various methods inside `SinkRecordDescriptor` we can simply call into the `columnFilter.matches()` function for both cases where a column filter is specified or if the user did not define any because we always return a positive match in the latter case.